wp-content/themes/missilethreat/inc/template-tags.php

```diff
@@ -409,4 +409,106 @@ function missiledefense_system_terms() {
 			printf( '%1$s', $html);
 		}
 	}
-endif;
\ No newline at end of file
+endif;
+
+if ( ! function_exists( 'missiledefense_defsys_us' ) ) :
+	/**
+	 * Returns links to U.S. Defense Systems and their icons.
+	 *
+	 */
+	function missiledefense_defsys_us() {
+		$args = array(
+			'post_type' => 'systems',
+			'numberposts' => -1,
+			'orderby' => 'post_title',
+			'order' => 'ASC',
+			'exclude' => array(4087, 4112, 4094, 4103, 4108, 7496, 7492, 7494, 7490)
+		);
+
+		$systems = get_posts( $args );
+
+		foreach ( $systems as $post) {
+		setup_postdata( $post );
+		$archive_name = get_the_title($post->ID);
```

Comment:
I would recommend making this output a template part or function, since you reuse it 3 times. If you have to make edits in the future, you'd only have to do it one place. Something like this:

```
foreach ($systems as $post) {
  setup_postdata( $post );
  template part actor;
}
```